platform/mbed_alloc_wrappers.cpp

```diff
@@ -329,6 +329,122 @@ extern "C" void $Sub$$free(void *ptr) {
 #endif // #if defined(MBED_MEM_TRACING_ENABLED) || defined(MBED_HEAP_STATS_ENABLED)
 
 /******************************************************************************/
+/* IAR memory allocation wrappers                                           */
+/******************************************************************************/
+#elif defined(__ICCARM__) // #if defined(TOOLCHAIN_GCC)
```

Comment:
do we need this lingering comment?

Reply:
@sg- Nah, just kept to have consistency.. Will remove